graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/modules/data/PageableAdapter.java

```diff
@@ -47,7 +47,7 @@ public Object getDefaultValue() {
 
     @Override
     public GraphQLInputObjectField transformInputField(GraphQLInputObjectField field, InputField inputField, OperationMapper operationMapper, BuildContext buildContext) {
-        if (field.getName().equals("pageSize") && field.getDefaultValue() == null && !(field.getType() instanceof GraphQLNonNull)) {
+        if (field.getName().equals("pageSize") && field.getInputFieldDefaultValue() == null && !(field.getType() instanceof GraphQLNonNull)) {
```

Comment:
should it not be `getInputFieldDefaultValue().getValue()`?

Reply:
@kaqqao did you spot this before merge? ⬆️ might be worth checking that particular functionality before any release. Thanks. 🙇‍♂️

Reply:
Yup, I fixed this in master directly (not sure why I did it that way, to be honest 🙁). Thanks, @indentia, for noticing this :)

Reply:
Awesome! Excellent collaboration. 💪